spec/src/main/asciidoc/app-programming-model.adoc

```diff
@@ -97,7 +97,7 @@ The following Annotations exist, see below for common fields:
 |@Counted | M, C, T | Denotes a counter, which counts the invocations of the annotated object. | MetricUnits.NONE
 |@Gauge   | M | Denotes a gauge, which samples the value of the annotated object.  | _no default_, must be supplied by the user
 |@Metric  | F, P | An annotation that contains the metadata information when requesting a metric to be injected. | MetricUnits.NONE
-|@Timed   | M, C, T | Denotes a timer, which tracks duration of the annotated object. | MetricUnits.NANOSECONDS
+|@Timed   | M, C, T | Denotes a timer, which tracks duration of the annotated object. | MetricUnits.SECONDS
```

Comment:
I'd suggest we leave the unit as NANOSECONDS and add a footnote that says that changing the units of a Timer has no effect.